modules/os2forms_forloeb/src/MaestroHelper.php

```diff
@@ -466,6 +466,21 @@ public function renderNotification(WebformSubmissionInterface $submission, strin
       ?? $data[$recipientElement]
       ?? NULL;
 
+    // Handle composite elements.
+    if ($recipient === NULL) {
+      if (str_contains($recipientElement, '__')) {
+        $keys = explode('__', $recipientElement);
+        $result = $data;
+        foreach ($keys as $key) {
+          if (array_key_exists($key, $result)) {
+            $result = $result[$key];
+          }
+        }
```

Comment:
https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Utility%21NestedArray.php/function/NestedArray%3A%3AgetValue/10 is your friend.

Reply:
Used `NestedArray`.